frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -996,7 +996,7 @@ export const ClusterSettingsPage: React.FC<ClusterSettingsPageProps> = ({ match
     },
     {
       href: 'globalconfig',
```

Comment:
Wondering if we should change the route to `config` and add a redirect from `globalconfig`?  Related, do we rename `GlobalConfigPage`?  @spadgett, opinions?

Reply:
I have an optional commit including the rename (went with route: `configuration` so it matches the tab label), but discovered a redirect isn't easily achievable with the way tabs work.  What do you think, @spadgett?

Reply:
Sorry for the late response. If redirects don't work well, I'd probably just keep the existing URL. Better not to break links / bookmarks.

Reply:
Sounds good.  I've removed the rename commit.  That means the file name, component names, and route all retain their original "global" designations.